### ISSUE TITLE
Doesnt work on Notepad++ 7.5.4

### ISSUE THREAD

**Issue by originalnoob** [wakatime/notepadpp-wakatime#28]
Installed it added api key didnt work. upgrade it try again didnt nothing work.

**Comment by Dave-Morton** [comment]
I don't know if my problem is the same as @originalnoob , but I have the same symptoms. I, however, have some additional information, so I thought I would share. I'm sharing the WakaTime log that was generated. From the log, it seems that some files were unable to be downloaded, though the reason this is so isn't clear in the log entries.
[WakaTime.log](https://github.com/wakatime/notepadpp-wakatime/files/1778676/WakaTime.log)

This is on a Windows 10 Pro (64 bit) OS, using Notepad++ 32 bit, version 7.5.5, with the latest version of the plugin manager. I'm wondering what I can do next, so any insights would be appreciated.

**Comment by ycczchina** [comment]
Same here

**Comment by gillespieza** [comment]
I've got nothing - nothing being written to the logs at all. In fact, when I went to the directory the log file was meant to be in, there was no log at all. I created a blank file, hoping maybe it would work then, but nada.

If I type `wakatime` in the command line, I get an error - maybe the CLI is not being downloaded?

I tried running Notepad++ as administrator and reinstalling the Wakatime plugin, but nothing changed.

EDIT: I'll create a separate issue, because I don't have quite the same symptoms, so this might be a separate issue.
